scripts/local.js

```diff
@@ -0,0 +1,60 @@
+const webdriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+var browserstack = require('browserstack-local');
+
+//creates an instance of Local
+var bs_local = new browserstack.Local();
+
+// replace <browserstack-accesskey> with your key. You can also set an environment variable - "BROWSERSTACK_ACCESS_KEY".
+var bs_local_args = { 'key': ACCESS_KEY };
+
+// starts the Local instance with the required arguments
+bs_local.start(bs_local_args, function() {
+  console.log("Started BrowserStackLocal");
+  // check if BrowserStack local instance is running
+  console.log(bs_local.isRunning());
```

Comment:
Not using this check for anything, let's add something like a try catch and only start the session if local is up.